libmscore/property.cpp

```diff
@@ -160,32 +165,32 @@ static const PropertyData propertyList[] = {
       { P_ID::REPEAT_MEASURE,      true,  0,               P_TYPE::BOOL    },
       { P_ID::REPEAT_JUMP,         true,  0,               P_TYPE::BOOL    },
 
-      { P_ID::MEASURE_NUMBER_MODE, false, "measureNumberMode",     P_TYPE::INT     },
-
-      { P_ID::GLISS_TYPE,          false, 0,                       P_TYPE::INT     },
-      { P_ID::GLISS_TEXT,          false, 0,                       P_TYPE::STRING  },
-      { P_ID::GLISS_SHOW_TEXT,     false, 0,                       P_TYPE::BOOL    },
-      { P_ID::DIAGONAL,            false, 0,                       P_TYPE::BOOL      },
-      { P_ID::GROUPS,              false, 0,                       P_TYPE::GROUPS    },
-      { P_ID::LINE_STYLE,          false, "lineStyle",             P_TYPE::INT       },
-      { P_ID::LINE_COLOR,          false, 0,                       P_TYPE::COLOR     },
-      { P_ID::LINE_WIDTH,          false, 0,                       P_TYPE::SPATIUM   },
-      { P_ID::LASSO_POS,           false, 0,                       P_TYPE::POINT_MM  },
-      { P_ID::LASSO_SIZE,          false, 0,                       P_TYPE::SIZE_MM   },
-
-      { P_ID::TIME_STRETCH,        false, "timeStretch",           P_TYPE::REAL      },
```

Comment:
Only thing I changed is the formatting (alignment), I'm not sure how the
value got changed, but I changed it back to what it was.

On Tue, Aug 23, 2016 at 9:54 PM, Marc Sabatella <notifications@github.com>
wrote:

> In libmscore/property.cpp
> <https://github.com/musescore/MuseScore/pull/2789#discussion_r75901595>:
>
> > @@ -160,32 +165,32 @@ static const PropertyData propertyList[] = {
> >        { P_ID::REPEAT_MEASURE,      true,  0,               P_TYPE::BOOL    },
> >        { P_ID::REPEAT_JUMP,         true,  0,               P_TYPE::BOOL    },
> >
> > -      { P_ID::MEASURE_NUMBER_MODE, false, "measureNumberMode",     P_TYPE::INT     },
> > -
> > -      { P_ID::GLISS_TYPE,          false, 0,                       P_TYPE::INT     },
> > -      { P_ID::GLISS_TEXT,          false, 0,                       P_TYPE::STRING  },
> > -      { P_ID::GLISS_SHOW_TEXT,     false, 0,                       P_TYPE::BOOL    },
> > -      { P_ID::DIAGONAL,            false, 0,                       P_TYPE::BOOL      },
> > -      { P_ID::GROUPS,              false, 0,                       P_TYPE::GROUPS    },
> > -      { P_ID::LINE_STYLE,          false, "lineStyle",             P_TYPE::INT       },
> > -      { P_ID::LINE_COLOR,          false, 0,                       P_TYPE::COLOR     },
> > -      { P_ID::LINE_WIDTH,          false, 0,                       P_TYPE::SPATIUM   },
> > -      { P_ID::LASSO_POS,           false, 0,                       P_TYPE::POINT_MM  },
> > -      { P_ID::LASSO_SIZE,          false, 0,                       P_TYPE::SIZE_MM   },
> > -
> > -      { P_ID::TIME_STRETCH,        false, "timeStretch",           P_TYPE::REAL      },
>
> Be sure not to overlook this! It appears you inadvertently changed stuff
> that probably breaks things here. There was no reason any of this code
> should have shown as a diff.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/musescore/MuseScore/pull/2789/files/b875e9fe5be4d970b59c6721f2e3a62469df0af3#r75901595>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGf7yoyx2rCJmueGKljptjU_SiLwnrxEks5qix6wgaJpZM4JpFpz>
> .
>



-- 
Ruchit Agrawal.
IIIT Hyderabad.